src/templates/article.jsx

```diff
@@ -1,40 +1,106 @@
 import React, {Fragment} from "react";
-import { Link, graphql } from "gatsby";
+import { Link as GatsbyLink, graphql } from "gatsby";
 import { MDXRenderer } from "gatsby-plugin-mdx";
 import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
+import { makeStyles } from '@material-ui/core/styles';
 
 import SEO from "src/components/SEO";
+import TagChip from "src/components/tagchip";
 import Layout from "src/layout";
 
+const useStyles = makeStyles((theme) => ({
+  avatarContainer: {
+    display: "flex",
+    "& > *": {
+      marginRight: theme.spacing(1),
+    },
+  },
+  link: {
```

Comment:
Can we do this for links universally, instead of just specifying it for some links?

Reply:
Reiterating our earlier conversation: let's make a link component to reuse throughout the website.